### PR TITLE
Lint Camunda Cloud 1.x BPMN Diagrams

### PR DESCRIPTION
**Pull request by philippfromme**
https://github.com/camunda/bpmnlint-plugin-camunda-compat is used to lint Camunda Cloud 1.x BPMN diagrams. Existing UI is used to display linting results.

electron_qQxNbMcG1n

Related to https://github.com/camunda/camunda-modeler/issues/2464.

### CODE REVIEW

**Comment by nikku** [comment]
I guess this is ready for review?

**Comment by philippfromme** [comment]
👍🏻

File: client/src/app/tabs/cloud-bpmn/linting/BpmnLinter.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+import { Linter } from 'bpmnlint';
+
+import BpmnModdle from 'bpmn-moddle';
+
+import { isString } from 'min-dash';
+
+import linterConfig from '../../.bpmnlintrc';
+
+const moddle = new BpmnModdle();
```
Comment (nikku) [inline client/src/app/tabs/cloud-bpmn/linting/BpmnLinter.js:19]: Shouldn't we instantiate moddle with `camunda` and `zeebe` namespace awareness?

Alternative: Don't lint files for the moment (throw a `not supported` error).

Comment (philippfromme) [reply client/src/app/tabs/cloud-bpmn/linting/BpmnLinter.js]: Fixed (cf. https://github.com/camunda/camunda-modeler/pull/2610/commits/2ee2552494cb2fb79a523b5a5a0314f032e9ebab).

**Review by nikku — APPROVED** [review]

**Comment by nikku** [comment]
@andreasgeier We'll ask for you're review of the whole feature once we've worked on all things described in https://github.com/camunda/camunda-modeler/issues/2464.

**Pull request merged**